README.md

```diff
@@ -242,15 +179,17 @@ times before querying the next recommendations.
 
 ```python
 df = campaign.recommend(batch_quantity=3)
+print(df)
 ```
 
-For a particular random seed, the result could look as follows:
+> ```python
+>    Granularity  Pressure[bar]    Solvent
+> 15      medium            1.0  Solvent D

```

Comment:
For me (when building locally), that output looks a bit weird:
<img width="1153" alt="image" src="https://github.com/emdgroup/baybe/assets/84016733/3bc70980-db5b-4829-9426-817600041ab3">
Might need to figure out why there is that blue box (which is also present in light mode)

Reply:
Removed all `>`